Assets/Scripts/PlayerControls.cs

```diff
@@ -0,0 +1,58 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerControls : MonoBehaviour
+{
+    public float Speed = 10f;
+    public float JumpingPower = 10f;
+
+    private float horizontal;
+    private bool isFacingRight = true;
+
+
+    [SerializeField] private Rigidbody2D rb;
+    [SerializeField] private Transform detectGround;
+    [SerializeField] private LayerMask groundLayer;
+    // Update is called once per frame
+    void Update()
+    {
+        // Vector2 input = new Vector2(Input.GetAxis("Horizontal"), Input.GetAxis("Vertical"));
+        // transform.Translate(input * Speed * Time.deltaTime);
+        horizontal = Input.GetAxis("Horizontal");
+
+        if(Input.GetButtonDown("Jump") && isGrounded())
+        {
+            rb.velocity = new Vector2(rb.velocity.x, JumpingPower);
+        }
+
+        if (Input.GetButtonDown("Jump") && rb.velocity.y > 0f)
```

Comment:
Use the new input system that unity has https://docs.unity3d.com/Manual/com.unity.inputsystem.html
Ask @Emiliontti since he has experience setting that up